packages/ttpassgen/PKGBUILD

```diff
@@ -0,0 +1,31 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=TTPassGen
+pkgver=133.a06d99d
+pkgrel=1
+pkgdesc='is a highly flexiable and scriptable password dictionary generator base on Python, you can easily use various rules to generate the desired combination of words.'
```

Comment:
Begins with a capitalized sentence and omitting the tool name or saying `is a`: eg: `Highly flexible and scriptable password dictionary generator.`.

Don't put the language, keep it short and in general try not to re-use the tool disclaimer unless it's well written.

By the way the upstream desc has a typo:

```diff
-flexiable
+flexible
```

Reply:
Corrected spelling.